### PR TITLE
Add a service reference card

### PR DESCRIPTION
**Pull request by gregcorbett**
as requested by EMT

### CODE REVIEW

File: doc/service_reference_card.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Service Reference Card Template
```
Comment (tofu-rocketry) [inline doc/service_reference_card.md]: Guess this should be renamed to some like "Accounting Service Reference Card".

File: doc/service_reference_card.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  * atd
+  * apeldbloader-cloud
+
+* Init commands (services started on container creation)
+  * service httpd start
+  * service crond start
+  * service atd start
+  * service apeldbloader-cloud start
+
+* Configuration files location
```
Comment (tofu-rocketry) [inline doc/service_reference_card.md]: "Configuration file locations"

File: doc/service_reference_card.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  * /etc/httpd/conf.d/apel_rest_api.conf - forces SSL
+  * /etc/httpd/conf.d/ssl.conf - configures SSL
+  * /etc/apel/cloudloader.cfg - configuration for the loader
+  * /etc/apel/cloudsummariser.cfg - configuration for the summariser
+
+* Logfile locations
+  * /var/log/cloud/loader.log - log file for the loader
+  * /var/log/cloud/summariser.log - log file for the summariser
+
+* Open ports
```
Comment (tofu-rocketry) [inline doc/service_reference_card.md]: Language in this section still seems a bit clunky (from the admin doc).

Maybe reword the second half of the port 80 and 443 points to something like "the Apache server [/for HTTPS] forwards traffic to the APEL server for a particular URL pattern [true?]."

File: doc/service_reference_card.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  * 80 - used by the APEL server service, by the Apache server and forwarded to the APEL server.
+  * 443 - used by the APEL server service, by the Apache server (for HTTPS) and forwarded to the APEL server.
+  * 3306 - used by the APEL server service and the MySQL service.
+
+* Possible unit test of the service
+  * Unit tests conducted during development via [TravisCI](https://travis-ci.org/apel/rest)
```
Comment (tofu-rocketry) [inline doc/service_reference_card.md]: "Continuous integration tests conducted..."

File: doc/service_reference_card.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  * For end to end testing, data could be POSTED to the endpoint, summarised and the retrieved from the GET endpoint
```
Comment (tofu-rocketry) [inline doc/service_reference_card.md]: "then retrieved"

File: doc/service_reference_card.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+* Security information
+  * Access control Mechanism description (authentication & authorization)
+    * X.509 certificates for POST requests
+    * IAM tokens for GET requests
+  
+  * How to block/ban a user
+    * To ban users accessing summaries, remove them from `ALLOWED_FOR_GET` in `/var/www/html/apel_rest/settings.py`
+  
+  * Network Usage
+    * None
```
Comment (tofu-rocketry) [inline doc/service_reference_card.md]: Can this and Firewall not have something in them saying what external thing _is_ responsible for these?

File: doc/service_reference_card.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+  * /etc/cron.d/cloudsummariser - runs the summariser
+  * /etc/cron.d/IGTF-bundle-update - updates the IGTF bundle on the first day of the month
+
+* Security information
+  * Access control Mechanism description (authentication & authorization)
+    * X.509 certificates for POST requests
+    * IAM tokens for GET requests
+  
+  * How to block/ban a user
+    * To ban users accessing summaries, remove them from `ALLOWED_FOR_GET` in `/var/www/html/apel_rest/settings.py`
```
Comment (tofu-rocketry) [inline doc/service_reference_card.md]: No method to block POSTing?

File: doc/service_reference_card.md (hunk 3 above)
Comment (gregcorbett) [inline doc/service_reference_card.md]: I have changed it to 

* Open ports
  * 80 - all traffic to this port is forwarded to port 443 by the Apache server.
  * 443 - the Apache server forwards (HTTPS) traffic to the APEL server, which returns a Django view for recognised URL patterns.
  * 3306 - used by the APEL server service and the MySQL service to communicate with each other.



Shall I change in admin.md as well?

File: doc/service_reference_card.md (hunk 4 above)
Comment (gregcorbett) [inline doc/service_reference_card.md]: done

File: doc/service_reference_card.md (hunk 5 above)
Comment (gregcorbett) [inline doc/service_reference_card.md]: done
